src/components/ListControls/ListControls.vue

```diff
@@ -1,32 +1,35 @@
 <template>
```

Comment:
I noticed that you are using scss syntax in your styles but you haven't added the "lang='scss''" attribute to the style tag